bigbluebutton-html5/imports/api/notification/index.js

```diff
@@ -0,0 +1,2 @@
+//TODO - send the inserted notification to a notification component.
+export default new Mongo.Collection(null);
```

Comment:
For me, since its an database connection, should be under imports/api, based on the meteor documentation, where we expect to find it like the other connections.